### ISSUE TITLE
[FEATURE] Add oled black theme

### ISSUE THREAD

**Issue by stacsk** [vishaltelangre/NerdCalci#198]
### Problem

Looking at any color other than pitch black tires my eyes.

### Desired solution

Oled theme would improve accessibility since it reduces eye strain. Would you consider adding a pitch black theme?

### Additional context

Thanks very much for this beautiful app. <3
